src/components/ui/RangeSlider.js

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import Slider from '@mui/material/Slider';
+
+function valuetext(value) {
+    return `${value}°C`;
+}
+
+const minDistance = 0;
+
+export default function RangeSlider() {
+    const [value1, setValue1] = React.useState([20, 37]);
+
+    const handleChange1 = (event, newValue, activeThumb) => {
+        if (!Array.isArray(newValue)) {
+            return;
+        }
+
+        if (activeThumb === 0) {
+            setValue1([Math.min(newValue[0], value1[1] - minDistance), value1[1]]);
+        } else {
+            setValue1([value1[0], Math.max(newValue[1], value1[0] + minDistance)]);
+        }
+    };
+
+    const [value2, setValue2] = React.useState([20, 37]);
+
+    const handleChange2 = (event, newValue, activeThumb) => {
+        if (!Array.isArray(newValue)) {
+            return;
+        }
+
+        if (newValue[1] - newValue[0] < minDistance) {
+            if (activeThumb === 0) {
+                const clamped = Math.min(newValue[0], 100 - minDistance);
+                setValue2([clamped, clamped + minDistance]);
+            } else {
+                const clamped = Math.max(newValue[1], minDistance);
+                setValue2([clamped - minDistance, clamped]);
+            }
+        } else {
+            setValue2(newValue);
+        }
+    };
+
+    return (
+            <Slider
+                getAriaLabel={() => 'Minimum distance'}
+                value={value1}
+                onChange={handleChange1}
+                valueLabelDisplay="auto"
+                getAriaValueText={valuetext}
+                disableSwap
+            />
+    );
+}
```

Comment:
I guess you tried to create a general RangeSlider but then some kind of issue appeared (you didn't know how to change the state of the parent component? (you can pass a function as a prop)). Just speculating.. However, since we are not using the slider anywhere else in our project, it is fine just having the slider in the PreferencePage, so you can delete that RangeSlider if not used.

Reply:
agreed. I deleted this i thought. 
![image](https://user-images.githubusercontent.com/91139250/168470499-2f6330f3-9d8c-40bb-ac83-89c74c5b284b.png)
also count find it for me, am i looking in the wrong place?

Reply:
you're looking at the css files, I mean the rangeslider in the /component/ui directory

Reply:
just saw myself that i am dumb
was looking at css and not js files